eclipse/sw360
# feat(ObligationDataModel): Changes in Obligation data model

> Changes in Obligation data model 
>   * Renamed struct Obligations to struct Obligation in License.thrift
>   * Renamed existing obligationType to obligationLevel
>   * Created new obligationType field which has Permission,Risk,Exception,Restriction as options
>   * Fixed adding obligation in licenses tab

> * Which issue is this pull request belonging to and how is it solving it? (*#982*, *#959*)
> * Did you add or update any new dependencies that are required for your change? - No

### How To Test?
> * Run migration scripts - old data should be migrated and functional in Project administrator ,Clearing Report, Obligation View
> * No unused design doc should be left behind
> * Add Obligation from Obligation View and License View and verify its functionality

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
